### ISSUE TITLE
"Package inclusion" text

### ISSUE THREAD

**Issue by fridex** [gofed/gofed-web#8]
A user can ask for a package inclusion, see:

http://209.132.179.123/request

We should make a very simple explanation for a user what's going on and how to use this form, e.g.:

> If you wish to include a package you can submit a request here. Please state all the necessary details in order to provide as detailed information as possible. Most of all, write down all reasons why should be your package included (e.g. dependencies, ...). Also specify a URL to github.com or bitbucket.org to requested project (or git/mercurial repo). This will help us to consider package inclusion and review source tree. Do NOT forget to add your e-mail, so we can contact you!

**Comment by fridex** [comment]
Same as https://github.com/gofed/gofed-web/issues/9#issuecomment-145510581, we should consider doing this more open (mailing list?) and be more open.

**Comment by ingvagabund** [comment]
The goal was to include all packages whose name does not start with golang- prefix. Packages with the prefix are automatically included when running scans on gofed-web. So I would rephrase it as:

"This form is meant for registration of golang projects whose package name does not start with "golang-" prefix, i.e. docker, kubernetes, runc. etc. Normally, automatic scans are run periodically on registered golang packages. Scans are designed to discover API backward incompatibilities, to create a dependency graph of golang projects packages and to collect other information that can be used to detect various issues which does not make the golang ecosystem healthy enough. If you register your project, you can benefit from the scans and be informed about updates of your project's dependencies (API breakages, new symbols, depricated packages, etc.) or about changes and recommended updates in your spec files.
To register your project please fill in your email, project's homepage (github.com, bitbucket.org, ...) and package name. In "Review request test" field you can provide additional information about the package (regular changes in API, active upstream, under development, etc.) or ask questions about scans. Your input and feedback is welcome."

What do you think?

**Comment by fridex** [comment]
OK, I will use it and commit to init data. Thanks!
